### PR TITLE
docs: add explanation ManyToMany with custom properties

### PR DESCRIPTION
**Pull request by maxsommer**
Since I myself ran into the issue and had to rewrite my code based on this solution I hope it may be helpful for others as well 😊

### CODE REVIEW

**Comment by vlapo** [comment]
Hi @maxsommer. Thank you for your contribution. We have small note in FAQ section about this topic. https://typeorm.io/#/faq/how-do-i-add-extra-columns-into-many-to-many-junction-table So feel free extend this section of docs and add link from https://typeorm.io/#/relations-faq and https://typeorm.io/#/many-to-many-relations

About your example. Please use more descriptive names of entities. Maybe some real-life scenario like relation between Post and Tag and extra column with `createdAt` or something like that. Such a case is easier to understand.

image

**Comment by maxsommer** [comment]
Alright, that's good to know – I did not notice that. I'll follow your advice and extend a little further and make the example more graspable. Thanks for the very quick and constructive feedback 😊

File: docs/many-to-many-relations.md [hunk 1]
```diff
@@ -169,3 +169,40 @@ const categoriesWithQuestions = await connection
...
+    @ManyToOne(type => Category, category => category.postToCategories)
+    public category!: Category;
+}
+```
+
+Additionally you will have to add a relationship like the following to `Post` and `Category`:
+
+```typescript
+@OneToMany((type) => PostToCategory, (postToCategory) => postToCategory.post)
+public postToCategories!: number;
```
Comment (vlapo) [inline docs/many-to-many-relations.md]: `PostToCategory[]` instead of `number` ?
```ts
public postToCategories!: PostToCategory[];
```

**Comment by maxsommer** [comment]
Thank you! Was my first open source PR so I'm very happy🙃

**Pull request merged**